src/System.Collections.Concurrent/src/System/Collections/Concurrent/ConcurrentQueue.cs

```diff
@@ -900,7 +901,8 @@ internal int High
                 {
                     //if _high > SEGMENT_SIZE, it means it's out of range, we should return
                     //SEGMENT_SIZE-1 as the logical position
-                    return Math.Min(_high, SEGMENT_SIZE - 1);
+                    var high = _high; // jit won't inline Math.Min if passed a volatile
```

Comment:
Nit: please name the type instead of using var, here and above

Also, I'm curious, does this make a measurable difference on throughout? I'd sort of expect this to be in the noise, though I'm fine with the change even if it is.